config/propeller.toml

```diff
@@ -5,6 +5,7 @@ ClientHeader = "x-user-id"
 EnableDeviceSupport = true
 DeviceHeader = "x-device-id"
 DeviceAttributeHeaders = ["x-os", "x-os-version"]
+EnableDebuggingHandlers = true
```

Comment:
can we call this `EnableProfilingHandlers` which is a more specific and apt term here?

Reply:
done